### PR TITLE
took out icons from file since they are in react now as svgs

### PR DESCRIPTION
**Pull request by sbuckingham89**
Took out just the portion related to counselor years.

### CODE REVIEW

File: src/assets/crisistextline/sass/partials/02-elements/_icons.scss [hunk 1]
```diff
@@ -34,18 +34,6 @@
     text-align: center;
 }
 
-$user-icon-baby-chick: "🐣";
-$user-icon-year-1: "🌟";
-$user-icon-year-2: "💫";
-$user-icon-year-3: "✨";
-$user-icon-year-4: "⚡️";
-$yearStarList: $user-icon-year-1 $user-icon-year-2 $user-icon-year-3 $user-icon-year-4;
-@mixin counselor-year-star($year, $element) {
```
Comment (morrismatts) [inline src/assets/crisistextline/sass/partials/02-elements/_icons.scss:43]: Looks like `counselor-year-star` and `user-icon-baby-chick` are used in _sidebar.scss, so you'll probably need to remove those references too. I'm not sure how Sass deals with missing mixins/variables, but I'd guess it doesn't do great (and would be good to clean up even if it's not strictly required)

**Review by morrismatts — CHANGES_REQUESTED** [review]
I think there are a couple of extra references you'll need to remove on this one, but lmk if I'm just missing something :)

File: src/assets/crisistextline/sass/partials/02-elements/_icons.scss (hunk 1 above)
Comment (sbuckingham89) [reply src/assets/crisistextline/sass/partials/02-elements/_icons.scss:43]: @morrismatts - Okay, just took that out if you want to take another look. Thanks!

**Review by morrismatts — APPROVED** [review]
Nice! Look at all that sweet code deletion!

**Pull request merged**